src/index.pug

```diff
@@ -225,3 +225,31 @@ html
         li.list-group-item Hello, I'm a list item.
 
     footer
+      .container
+        ul.footer-links 
+          li 
+            a(href="#" class="info") Info
```

Comment:
Can you namespace these classes. There's a pretty good chance for a conflict without one when this footer is added to other sites. Something like `footer-link-info`, `footer-link-email`, etc.